### ISSUE TITLE
Install issues on macOS Monterey 12.6 (M1 Pro)

### ISSUE THREAD

**Issue by usrGabriel** [cirruslabs/tart#266]
Hello!
I was going through the steps to install tart via Brew, but I've been getting the following message on each attempt: "Error: Tart only works on Apple Silicon!", and I am running an M1 Pro machine on macOS Monterey 12.6, so I'm wondering if there might be a bug in the installer?

Here's a screenshot of the error as well as system specs.
![Tart install issue](https://user-images.githubusercontent.com/26032081/194795498-46da997c-2a53-4fbe-b790-3353de1795dc.png)

I have tried the following to potentially fix the issue, but no luck:

- Ran 'brew upgrade' to make sure everything is up to date on Brew packages
- Ran 'brew doctor' to check for related issues, none were related to tart
- Rebooted the machine completely and tried to run the install command again

Can you confirm if this is an actual issue or if maybe I am not doing something correctly?
Many thanks!

**Comment by edigaryev** [comment]
> Can you confirm if this is an actual issue or if maybe I am not doing something correctly?

Is it possible that you're running Homebrew under Rosetta 2? For example:

```
% arch -x86_64 brew install tart
Error: Tart only works on Apple Silicon!
```

**Comment by usrGabriel** [comment]
Thanks for the info, I see that my whole Homebrew install was in fact in Rosetta 2 since I restored from a backup which had the initial Homebrew environment set up in an Intel machine. 

I made a Brew file backup and completely reinstalled Brew for ARM instead, them reimported everything, and then installed tart successfully!

![Tart install fixed](https://user-images.githubusercontent.com/26032081/194957797-20fe3bb7-6468-473b-836e-04413d06100b.png)

Thanks for your help!
